dNet/ClientPackets.cpp

```diff
@@ -160,19 +165,30 @@ void ClientPackets::HandleClientPositionUpdate(const SystemAddress& sysAddr, Pac
 				vehiclePhysicsComponent->SetDirtyVelocity(velocityFlag);
 				vehiclePhysicsComponent->SetAngularVelocity(angVelocity);
 				vehiclePhysicsComponent->SetDirtyAngularVelocity(angVelocityFlag);
-
-				EntityManager::Instance()->SerializeEntity(possassableEntity);
-
-				hasVehicle = true;
+			} else {
+				// Need to get the mount's controllable physics
+				auto* pcomp = possassableEntity->GetComponent<ControllablePhysicsComponent>();
```

Comment:
Use the full component name with leading lowercase letter here.

Reply:
was this way when i found this code, but do agree and will fix